source/application/DbgHelp/DbgHelpStackWalkCallback.h

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (c) 2016- Hourglass Resurrection Team
+ * Hourglass Resurrection is licensed under GPL v2.
+ * Refer to the file COPYING.txt in the project root.
+ */
+
+#pragma once
+
+#define WIN32_LEAN_AND_MEAN
+#include <Windows.h>
+
+#include <string>
+
+class DbgHelpStackWalkCallbackPrivate;
+
+class DbgHelpStackWalkCallback

```

Comment:
Prevents leaking the DIA headers into the main code, they're heavy and intrusive for the IntelliSense.